app/views/entities/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-col justify-center h-full">
+  <div class="bg-main flex justify-between text-white items-center h-12">
+    <%= link_to "<", groups_path, class: 'text-xl mx-2'  %>
+    <h1 class="font-proxima-bold">ENTITIES</h1>
+    <%= link_to ">", new_group_entity_path(@group_id), class: 'text-xl mx-2'  %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=a%20new%20category.-,Transactions%20page,-For%20a%20given), This page supposes to be called `Transactions `page, Please change this header to be  `Transactions` instead of `Entities`.

Reply:
Thanks @Omar-Muhamad  i am changing to match requeriments